cloudformation/ci.template

```diff
@@ -1,49 +0,0 @@
-{
-  "AWSTemplateFormatVersion" : "2010-09-09",
-  "Outputs": {
-    "AccessKeyId": {
-      "Value": { "Ref": "AccessKey" }
-    },
-    "SecretAccessKey": {
-      "Value": { "Fn::GetAtt": ["AccessKey", "SecretAccessKey"] }
-    }
-  },
-  "Resources": {
-    "AccessKey": {
-      "Type": "AWS::IAM::AccessKey",
-      "Properties": {
-        "UserName": { "Ref": "User" }
-      }
-    },
-    "User": {
-      "Type": "AWS::IAM::User",
-      "Properties": {
-        "Policies": [
-          {
-            "PolicyName": "upload-sdk-registry",
-            "PolicyDocument": {
-              "Statement": [
-                {
-                  "Action": ["s3:ListBucket"],
-                  "Effect": "Allow",
-                  "Resource": [
-                    "arn:aws:s3:::mapbox-api-downloads-production",
```

Comment:
we can remove these now? where is the least privilege defined now?

Reply:
otherwise this would just break release publishing, right?

Reply:
Yea, I think you're right. I tested this, but I didn't redeploy the template so the removal probably didn't take effect.